ops/testing.py

```diff
@@ -962,8 +960,8 @@ def add_relation_unit(self, relation_id: int, remote_unit_name: str) -> None:
 
         Example::
 
-          rel_id = harness.add_relation('db', 'postgresql')
-          harness.add_relation_unit(rel_id, 'postgresql/0')
+            rel_id = harness.add_relation('db', 'postgresql')
+            harness.add_relation_unit(rel_id, 'postgresql/0')
```

Comment:
This change is only for consistency since most docstring code samples in `ops/testing.py` uses 4 spaces instead of 2. They don't affect the rendered result by sphinx.